wordpress-mobile/WordPress-Android
# Issue/6927 site creation details screen

Fixes #6927 

~Needs #7028 to be merged first!~ Done now.

Introduces the Site Details screen, asking the user for the site title (mandatory) and the site tagline (optional).

To test:
1. With the app logged in, head over to MySite and tap on the "Switch Site" button
2. Tap on the "+" action on the toolbar
3. Select "Creation WordPress.com site" option from the popup
4. Tap on any of the category cards
5. Tap on any of the themes
6. Notice the Site Details screen and compare it with the design [on the ticket](https://github.com/wordpress-mobile/WordPress-Android/issues/6927)
7. Try out the input boxes and the "Next" button
8. Upon successful completion of the form, a Toast pops up with the inputted values for the site title and tagline
